### PR TITLE
vcpkg: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.es/common/vcpkg.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# vcpkg
+
+> Gestor de paquetes para librerías C/C++.
+> Nota: los paquetes no se instalan en el sistema. Para usarlos, necesitas decirle a tu sistema de compilación (por ejemplo CMake) que use `vckg`.
+> Más información: <https://learn.microsoft.com/en-us/vcpkg/>.
```
Comment (tricantivu) [inline pages.es/common/vcpkg.md:5]:
```suggestion
> Más información: <https://learn.microsoft.com/vcpkg/>.
```

[See `style-guide.md`](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#microsoft-learn-links)

File: pages.es/common/vcpkg.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# vcpkg
+
+> Gestor de paquetes para librerías C/C++.
+> Nota: los paquetes no se instalan en el sistema. Para usarlos, necesitas decirle a tu sistema de compilación (por ejemplo CMake) que use `vckg`.
```
Comment (tricantivu) [inline pages.es/common/vcpkg.md:4]:
```suggestion
> Nota: los paquetes no se instalan en el sistema. Para usarlos, necesitas decirle a tu sistema de compilación (por ejemplo CMake) que use `vcpkg`.
```

File: pages.es/common/vcpkg.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# vcpkg
+
+> Gestor de paquetes para librerías C/C++.
+> Nota: los paquetes no se instalan en el sistema. Para usarlos, necesitas decirle a tu sistema de compilación (por ejemplo CMake) que use `vcpkg`.
+> Más información: <https://learn.microsoft.com/vcpkg/>.
+
+- Construye y añade el paquete `libcurl` al entorno `vcpkg`:
```
Comment (tricantivu) [inline pages.es/common/vcpkg.md:7]:
```suggestion
- Construye y añade el paquete `libcurl` al entorno de `vcpkg`:
```

File: pages.es/common/vcpkg.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+- Construye y añade `zlib` usando la cadena de herramientas `emscripten`:
+
+`vcpkg install --triplet=wasm32-emscripten zlib`
+
+- Busca un paquete:
+
+`vcpkg search {{nombre_del_paquete}}`
+
+- Configura un proyecto CMake para utilizar los paquetes `vcpkg`:
```
Comment (tricantivu) [inline pages.es/common/vcpkg.md:19]:
```suggestion
- Configura un proyecto CMake para utilizar los paquetes de `vcpkg`:
```

**Review by tricantivu — APPROVED** [review]

**Pull request merged**